app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUpdate.kt

```diff
@@ -33,7 +32,7 @@ class BloodSugarEntryUpdate @AssistedInject constructor(
     return when (event) {
       is ScreenChanged -> next(model.screenChanged(event.type))
       is DatePrefilled -> next(model.datePrefilled(event.prefilledDate))
-      is BloodSugarMeasurementFetched -> noChange()
+      is BloodSugarMeasurementFetched -> onBloodSugarMeasurementFetched(model, event)
```

Comment:
The prefix `on` indicates that this is some form of callback method, while this is a method that returns a `Next`. I feel we can just drop the `on` prefix.

What do you think?

Reply:
Yeah 👍